Bottom: 554976292801198ea54943729b8babca57a70464
Top:    4bbea401a3440a3f867884ca7558af689fe82e60
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-10 20:23:23 +0000

Adjust board thickness to 3.6mm (For logo testing)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f37dffe..610ecb5 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -77,7 +77,7 @@
 
 static Coord board_thickness;
 #define HACK_BOARD_THICKNESS board_thickness
-//#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+//#define HACK_BOARD_THICKNESS MM_TO_COORD(3.6)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
